spec/factories/driver_errors.rb

```diff
@@ -4,6 +4,7 @@
     error_message "runtime error"
     page_class "MiBridges::Driver::BasePage"
     page_html "<html></html>"
+    driven_at DateTime.current
```

Comment:
do we need `driven_at` on here if we have a FK for the driver application and a created at timestamp for this error?

Reply:
We do because this can be a different time (for older errors) from the parent driver application. Older errors would not be updated along with a recently updated driver application.

I thought the same thing as I was working on this actually but when I considered (_if_ you referent the parent driver application record) that changing the driven at for a driver_application would change driver_errors from a day or two ago, the redundancy made sense.